src/pages/Landing/Landing.js

```diff
@@ -42,27 +49,61 @@ const Landing = () => {
 
   const availableStates = useMemo(() => allStates.map((state) => state.label), [allStates]);
 
+  // Styles for the menu items
```

Comment:
![weird styling](https://github.com/precision-sustainable-ag/dst-selector/assets/55995241/77accce2-1866-465f-aa1c-2e306adb4506)
this dropdown behaves weird sometimes like the above screenshot